### ISSUE TITLE
How to linear  update data's values  through set API for  'setOptions'  more times  in echarts.         

### ISSUE THREAD

**Issue by baokeyu123** [apache/echarts#14432]
### Version
next

### Steps to reproduce

       chartInstance.setOption({
            ...option,
            series: [

                {
                    type: 'line',
                    data: data,
                    smooth: true,
                    silent: true
                },]

### What is expected?
how to resolve this problems

### What is actually happening?
firstly  when i animationDuration in setOption. the chart's liner can linear paint.
however,when i want to update data through setOption again.   the chart'liner paint right now.
how to linear update data‘s values through second setOption or more times.
thanks.

---
please help me to resolve problem,thanks

**Comment by baokeyu123** [comment]
@leeight 当我通过setOption 动态push数据时，如何能更加匀速的更新非初始化Echart视图，谢谢

**Comment by plainheart** [comment]
To keep a linear update, You will need to set `animationDuration` as the same as the update interval.

**Comment by baokeyu123** [comment]
@plainheart In the future, the official API will be issued to solve the problem of dynamic data update animation?after all, through set interval to linear update paint is so trouble

thanks you very much

**Comment by plainheart** [comment]
Yes. I think it's necessary to make the update animation more smooth, not by setting the animation duration and interval.
